bin/wrap-fastframe

```diff
@@ -67,19 +68,40 @@ if rank == 0:
         night = os.path.basename(os.path.dirname(os.path.dirname(filename)))
         if (args.start <= night) & (night < args.stop):
             flavor = fits.getval(filename, 'FLAVOR')
+
             expid = fits.getval(filename, 'EXPID')
-            if flavor not in ('flat', 'science'):
+
+            if flavor not in tuple(args.exptypes.split(',')):
                 print('night {} expid {} is {}; skipping'.format(
                     night, expid, flavor))
+
+                logfile = filename.replace('simspec', 'fastframe').replace('.fits', '.log')
+
+                assert logfile != filename
+
+                if args.outdir:
+                  ##  If outdir provided, log files inherit /night/exposure structure of simspec files but in outdir.                                                                                          
+                  ##  Else placed in original dir. with simspecs.                                                                                                                                              
+
+                  logfile = logfile.replace(desisim.io.simdir(), args.outdir)
+                
+                try:
+                  os.makedirs(os.path.dirname(logfile), exist_ok=args.clobber)
+                except:
+                  print('Pass --clobber to replace previously created logfile for {}.'.format(expid))
```

Comment:
Please use 4-space indentation instead of 2-space indentation lines 82-91 (and elsewhere)